clients/fides-js/src/lib/consent-utils.ts

```diff
@@ -150,7 +150,7 @@ export const experienceIsValid = (
   if (
     !(
       effectiveExperience.privacy_notices &&
-      effectiveExperience.privacy_notices.length >= 0
+      effectiveExperience.privacy_notices.length > 0
```

Comment:
❗ another subtle change that is unrelated, but the tests only started failing here. this update makes intuitive sense to me though—the experience is only valid if there are privacy notices, and there are more than 0